Automattic/wp-calypso
# [WIP] State: Reduxify media/list-store

#### Changes proposed in this Pull Request

* State: add the `listStore` reducer to the media parent reducer
* State: add selectors for `listStore` including ordering [todo - blocked by `media.store` store reduxification]
* State: add `MEDIA_SET_QUERY` action [todo - still needs action function]

#### Testing instructions

* Verify all tests pass (no functional changes)

Note: This is the first PR of a series to remove the media list-store flux store. Also see #39379 for the complete media reduxification effort.
